lib/TunnelCluster.js

```diff
@@ -1,10 +1,10 @@
-const { EventEmitter } = require('events');
-const debug = require('debug')('localtunnel:client');
-const fs = require('fs');
-const net = require('net');
-const tls = require('tls');
+const { EventEmitter } = require("events");
```

Comment:
Keep it with single quotes to keep the project pattern and decrease the amount of PR changes.